src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java

```diff
@@ -494,14 +494,14 @@ public Claim saveClaimantResponse(
             );
 
             Claim existingClaim = toClaim(startEventResponse);
-            Claim updatedClaim = existingClaim.toBuilder()
-                .claimantResponse(response)
+            Claim.ClaimBuilder updatedClaim = existingClaim.toBuilder();
```

Comment:
This is a builder object now, name would be misleading herer. I don't see any need to change this file at all.